awx/ui_next/src/screens/Host/HostAdd/HostAdd.jsx

```diff
@@ -23,6 +27,10 @@ function HostAdd() {
       const { data: response } = await HostsAPI.create(values);
       history.push(`${url}/${response.id}/details`);
     } catch (error) {
+      // check for field-specific errors from API
+      if (error.response?.data && typeof error.response.data === 'object') {
+        throw error.response.data;
```

Comment:
This is probably where we'd want to start implementing the error handling code for field errors - Do you see that as something you'd like to include in this PR or something for a later date?